extensions/wikia/ContentReview/controllers/ContentReviewApiController.class.php

```diff
@@ -87,6 +128,18 @@ public function getLatestReviewedRevision() {
 		}
 	}
 
+	public function showTestModeNotificaion() {
+		$notification = wfMessage( 'content-review-test-mode-enabled' )->escaped();
+		$notification.= Xml::element(
+			'a',
+			[ 'id' => 'content-review-test-mode-disable', 'href' => '#' ],
+			wfMessage( 'content-review-test-mode-disable' )->escaped(),
+			false
```

Comment:
You shouldn't need to set the `$allowShortTag` option as you have content in your tag.